### PR TITLE
Updates for Atmel radio part II

### PR DESCRIPTION
**Pull request by herjulf**
More get/set_value fixes and get_obj RADIO_PARAM_LAST_PACKET_TIMESTAMP support.
modified:   cpu/avr/radio/rf230bb/halbb.c
modified:   cpu/avr/radio/rf230bb/rf230bb.c

### CODE REVIEW

File: cpu/avr/radio/rf230bb/rf230bb.c [hunk 1]
```diff
@@ -1519,6 +1545,14 @@ rf230_set_pan_addr(unsigned pan,
     PRINTF("\n");
   }
 }
+
+/* From ISR context */
+void
+get_last_rx_packet_timestamp(void)
```
Comment (g-oikonomou) [inline cpu/avr/radio/rf230bb/rf230bb.c]: This looks like a global symbol and probably needs renamed to `rf230bb_get_last_rx_packet_timestamp()` to match the naming convention.

**Comment by g-oikonomou** [comment]
Good stuff. It appears as though those new functions in `rf230bb.c ` could be static?

**Comment by herjulf** [comment]
Agree. I'll fix that.

**Comment by g-oikonomou** [comment]
@heipei  Robert, ready here?

**Comment by herjulf** [comment]
Hello, hmm, Do we miscommunicate or something I'm missing? 
The rename was done in  commit aed54573322f668333fbd66ffdf40fb893b37a8c for 11h ago.

**Comment by g-oikonomou** [comment]
Just checking whether this is all done or whether you were planning to add more :)

**Comment by herjulf** [comment]
Thanks. Ok. Understand. Well more will come but need to care some other duties for some days now.

**Comment by g-oikonomou** [comment]
Thanks!

**Pull request merged**